README.md

```diff
@@ -31,10 +33,27 @@ The easiest way to install Machinery is as an RPM from our
 [homepage](http://machinery-project.org) with
 the one-click-installer.
 
-### Development
+## Usage
+
+Machinery is a command-line tool. You can invoke it using the `bin/machinery`
+command. It accepts subcommands (similarly to `git` or `bundle`).
+
+To display a short overview of available commands and their descriptions, use
+the `help` command:
+
+    $ machinery help
+
+For more information about the commands, see
+[Machinery man page](http://machinery-project.org/manual.html).
+
+## Documentation
+* [User Documentation](http://machinery-project.org/manual.html)
+* [Developer Documentation](https://github.com/SUSE/machinery/tree/master/docs)
+
+## Development
 
 The following steps are only recommended if you want to work on the Machinery
-codebase or test the latest development changes.
+codebase or test the latest development changes. Read our [Developer Documents](https://github.com/SUSE/machinery/tree/master/docs) first, before you start contributing.
```

Comment:
Instead of adding it here, I would add a point 6. to the list at the end:

6. Contibute

Now that you have Machinery running from git on your machine you are ready to hack. If you would like to get some overview of architecture and design of Machinery have a look at our [developer documentation].

We are happy if you share your changes with us as pull requests. Read the [contribution guidelines] for details how to do that.